privx_api/privx_api.py

```diff
@@ -386,6 +402,27 @@ def create_user(self, user: dict) -> PrivXAPIResponse:
         response = self._http_post("userstore.users", body=user)
         return PrivXAPIResponse(response, 201)
 
+    def get_users(self) -> PrivXAPIResponse:
```

Comment:
Might be a bit misleading as this only returns local users. Could change the function name accordingly.

Reply:
Updated function name to get_local_user